app/views/posts/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2><%= @author.name %></h2>
+<h2>Username</h2>
+<p>Number of posts: x</p>
+<% @author_posts.each do |post| %>
+  <a class="card-link" href="<%= user_post_path(post.id) %>">
+    <p>Post 1</p>
+    <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Vitae recusandae soluta tempore molestiae eum nam veniam autem. Mollitia tenetur ad provident, quisquam eius velit recusandae accusamus architecto hic, neque deserunt.</p>
+    <p>Comments: x Likes: x</p>
+    <p>Username: Comments 1</p>
+    <p>Username: Comments 2</p>
+    <p>Username: Comments 3</p>
+    <p>Username: Comments 4</p>
+    <p>Username: Comments 5</p>
+  </a>
```

Comment:
- [ ] Kindly use the `recent_comments` method that you created in the `app/models/post.rb` file to render the recent comments in the `Posts#Index` page. 👍 😎